bundle/CMakeLists.txt

```diff
@@ -0,0 +1,97 @@
+if(NOT CMAKE_BUILD_TYPE)

```

Comment:
This is the bundle. It is an extension, and optional functionality: the CMake of serialbox does not require using the bundle in order to compile it. I recommend you to continue reading the rest of the tutorial and come back to this once you are finished with all the new CMake functionality

What does the bundle?
===================

It resolves the dependencies (recursively) of a CMake package. 
Serialbox defines here the sequence of dependencies before compiling itself. 
For each dependency package (let call it B), if ```bundle/CMakeLists.txt``` is provided, then the bundle of B will resolve its dependencies before continuing. 
Let assume the following DAG of dependencies: 
```
A -> B, CExt
B -> DExt
```
Then the bundle will compile the packages in the following order `DExt, CExt, B, A`